test/lib/usd/translators/testUsdExportUVSets.py

```diff
@@ -84,6 +87,51 @@ def setUpClass(cls):
         cmds.select("box.map[0:299]", r=True)
         cmds.polyEditUV(u=1.0, v=1.0)
 
+        # XXX: Although the UV sets on the "SharedFacesCubeShape" are stored in
+        # the Maya scene with a minimal number of UV values and UV shells, they
+        # seem to be expanded when the file is opened such that we end up with
+        # a UV value per face vertex rather than these smaller arrays of UV
+        # values with only the indices being per face vertex. As a result, we
+        # have to reassign the UVs just before we export.
```

Comment:
Does this ring any bells with anyone at Autodesk?

I attempted to author the UVs in the way I'd like to see them exported in 25e721b14927af1f8ae9528fb9ac793e1fa24d9b, but found that I needed to explicitly re-author them after opening the file in the test.

If you were to open the Maya scene file, run the `clear()`, `setUVs`(), and `assignUVs()` commands below and then save the file, there is no change to the mesh in the Maya scene file.

Any thoughts?

Reply:
Might be a missing freeze history. Are you seeing extra operators added to the mesh construction history in the save file?

Reply:
Thanks @JGamache-autodesk. There were some `polyMapDel` nodes in the Maya scene file, but those applied to the `OneMissingFaceCubeShape` and `OneAssignedFaceCubeShape`. I deleted all history in the scene and that cleaned up those shapes, so I pushed that in b94de4fe8de195b963dbdedc6879dcbacaf3d7f3, but it unfortunately didn't seem to affect `SharedFacesCubeShape`, so I couldn't take this workaround out of the test script.